lib/src/screens/send/send_page.dart

```diff
@@ -173,6 +173,9 @@ class SendPage extends BasePage {
                                         .headline
                                         .decorationColor),
                                 validator: sendViewModel.addressValidator,
```

Comment:
Better to make new validator which will use address validator and domain validator (create if not exists) and keep validation for address fields.